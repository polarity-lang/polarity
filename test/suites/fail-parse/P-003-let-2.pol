data Nat { Z, S(x: Nat)}

let x : Nat { let x : Nat := let y := Z; y ; S(x) }