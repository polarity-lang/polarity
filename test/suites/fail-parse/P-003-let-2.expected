P-003

  × Unexpected "let", expected "comatch", "(", "?", "\\", "_", "identifier", "number literal"
   ╭─[P-003-let-2.pol:3:30]
 2 │ 
 3 │ let x : Nat { let x : Nat := let y := Z; y ; S(x) }
   ·                              ───
   ╰────
P-003

  × Unexpected ":=", expected "(", ":"
   ╭─[P-003-let-2.pol:3:36]
 2 │ 
 3 │ let x : Nat { let x : Nat := let y := Z; y ; S(x) }
   ·                                    ──
   ╰────
